api/keto/acl/v1/check_service.proto

```diff
@@ -0,0 +1,71 @@
+syntax = "proto3";
+
+package keto.acl.v1;
+
+import "keto/acl/v1/acl.proto";
+import "google/api/client.proto";
+
+option go_package = "github.com/ory/keto/api/keto/acl/v1;acl";
+option csharp_namespace = "Ory.Keto.Acl.V1";
+option java_multiple_files = true;
+option java_outer_classname = "CheckServiceProto";
+option java_package = "sh.ory.keto.acl.v1";
+option php_namespace = "Ory\\Keto\\Acl\\V1";
+
+// The service that performs authorization checks
+// based on the stored Access Control Lists.
+service CheckService {
+  option (google.api.default_host) = "keto.exampleapis.com";
+  // Check performs an authorization check.
+  rpc Check(CheckRequest) returns (CheckResponse);
+}
+
+// The request for a CheckService.Check rpc.
+// Checks whether a specific subject is related to an object.
+message CheckRequest {
+  // The related object in this check.
+  Object object = 1;
+  // The relation this check.
+  string relation = 2;
+  // The concrete subject id to check.
+  string subject_id = 3;
```

Comment:
In case of a subject set, would this mean that it has to be encoded? Wouldn't it be better if this was of type `Subject`? I am not sure what the use case would be but it is really just a definition thing, so we can allow it without any drawbacks.

Reply:
Oh, sure!

I'll put that in, so one can `Check` on a `SubjectSet` (without expantion I guess).

What I just thought is that it might confuse API users if they see they can do a check on a `SubjectSet`.
- I mean one could understand that the semantic is that the server expands the `SubjectSet` to subject ids, therefor making a `Check` request on multiple subject ids. Like WHAT?! :D

Reply:
BTW do we also want to add an `skip_expand` bool to the CheckRequest.
- `skip_expand` defaults to `false` so expand checks are done by default (probably what the user expects?)
- setting `skip_expand` skips SubjectSet expanding and any rewrite rules?

Reply:
Isn't it the same if you expand a `SubjectSet` or not?
And isn't `skip_expand` the same as using read to see whether this exact tuple is stored?

Reply:
Yes, I mean the expand API (`2.4.5 Expand` in paper), whether if we simply add a `skip_expand` field to the `CheckRequest` that the client can explicitly say whether SubjectSets and Rewrite Rules apply for a check request.

Reply:
This would not be a breaking change so let's add it later. Tracked as https://github.com/ory/keto/issues/323